tests/channels/test_d0_to_ks_kp_km.py

```diff
@@ -3,8 +3,10 @@
 """
 
 import logging
+import dot_parser
 
 from expertsystem.ui import StateTransitionManager
+from expertsystem.topology.graph import DotGenerator
```

Comment:
Check import order (`pylint` check). It's best to activate `pre-commit`, see
https://pwa.readthedocs.io/projects/expertsystem/en/0.3.0-alpha/contribute.html#pre-commit